.github/workflows/dbdocs.yml

```diff
@@ -0,0 +1,67 @@
+name: "Build & Publish Database Documentation"
+
+on:
+  push:
+    branches:
+      - add-schema-spy-github-action
```

Comment:
should it also run on every push to production?

Reply:
oh and I guess this line would be removed after testing is done/ before it is merged to main/prod?

Reply:
Yea, and I think it's good time to do that now. Thank you for mentioning this! 089c4ad1d4c0f390e681a1e87952983f53278d76